src/PropertySuggester/Suggesters/SimpleSuggester.php

```diff
@@ -172,14 +172,15 @@ public function suggestByItem( Item $item, $limit, $minProbability, $context ) {
 
 				$entityId = $dataValue->getEntityId();
 
-				if ( !( $entityId instanceof Int32EntityId ) ) {
+				if ( !( $entityId instanceof ItemId ) ) {
 					throw new LogicException(
-						$entityId->getSerialization() . ' expected to be Int32EntityId'
+						"Property $numericPropertyId in wgPropertySuggesterClassifyingPropertyIds"
+						. ' does not have property type wikibase-item'
```

Comment:
I also think it's some problem to in some places say "property type" and other places "data type". I would be slightly confused if I saw the exception message (with "property type") and the above wording in the error logs.